_docs/development.md

```diff
@@ -25,23 +25,26 @@ You'll need the following to get started:
 ## Create temba user for PostgreSQL
 
 {% highlight bash %}
-$ createuser temba --pwprompt -d
+$ sudo apt-get install postgresql-client postgresql-server-dev-9.3 postgis*
+$ sudo -u postgres createuser temba --pwprompt -d
 Enter password for new role:
 Enter it again:
+$ sudo adduser temba
 {% endhighlight %}
 
 ## Create temba database, add PostGIS
 
 Create the database as temba user:
 {% highlight bash %}
-$ psql --user=temba postgres
+$ sudo -u temba psql --user=temba postgres
 postgres=> create database temba;
```

Comment:
Maybe, but that would be an issue with how you have Postgres installed. The problem is that a lot of these little things are dependent on your OS, how you have the DB installed and configured etc, that's why I'm hesitant to put in some of these changes because they are specific to your environment. (ie, these work as written on our environments)